docs/collecting-data/code-generation/commands/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Commands"
+sidebar_position: 4
+---
+
+:::info
+This page only summarizes the CLI commands and the options for each command. For details on which scenarios they can be used, you can go to the [Working with the CLI page](../using-the-cli/index.md).
+:::
+
+## Usage
+
+`snowtype [COMMAND] [OPTIONS] [CONTEXT-SPECIFIC-OPTIONS]`
+
+## Available CLI commands
+
+### `snowtype init`
+
+Initialize the setup of Snowtype code generation in a project. Creates the `snowtype.config.json` file.
+
+**Options**
+ -  `-f, --force` Override existing snowtype config. (default: false)
+ -  `-i, --organizationId` Organization ID.
+ -  `-t, --tracker` Tracker to use. [See available](../using-the-cli/index.md#available-trackerslanguages)
+ -  `-l, --language` Language to use. [See available](../using-the-cli/index.md#available-trackerslanguages)
+ -  `-o, --outpath` Output path.
+
+### `snowtype generate`
+
+Generates the code based on configuration on the `snowtype.config.json` file. Can generate/modify the `.snowtype-lock.json` file.
+
+### `snowtype update`
```

Comment:
This page is more of a command reference, no essence of user flow _(which is still being worked on)_. Since commands are, and probably be, just a few, I don't feel there is particular reason to order them based on something. Maybe just `init` and `generate` which would be the most interesting